client/src/app/app-routing.module.ts

```diff
@@ -1,19 +1,30 @@
 import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
+import {
+  RouterModule,
+  Routes,
+} from '@angular/router';
+
+import { RequireAuthenticatedGuard } from '@shared/authentication/require-authentication-guard';
 
 const routes: Routes = [
   {
     path: 'employees',
+    canActivate: [RequireAuthenticatedGuard],
```

Comment:
I was never very happy with all this repetition of guards when an entire app required authentication with the exception of login/forgot-password.

The best way to solve it that I came up with was putting all of the auth'd routes within their own module via a namespace in the URL, like `/a/`.

I think it might also be possible with a separate routing module with an empty route path `''`, but I'd have to test and see if there were issues with route resolution.